### PR TITLE
Added node-gyp prerequisites for Windows and other OSes

### PR DESCRIPTION
**Pull request by Vishal-beep136**
Added prerequisites for Windows OS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,21 @@ https://user-images.githubusercontent.com/10504365/192807716-6a4fcbc1-9dc8-4d3b-
...
+  - Python should be installed
+  - Visual C++ build tools workload for Visual Studio 2022
+
+  You can use **Chocolatey** Package Manager ([See Installation / Setup](https://docs.chocolatey.org/en-us/choco/setup/))
+  ```bash
+  choco install python visualstudio2022-workload-vctools -y
+  ```
+</details>
+
+```bash
```
Comment (karlhorky) [inline README.md:48]:
````suggestion
### Prerequisites

1. Set up ESLint following [the `typescript-eslint` Getting Started docs](https://typescript-eslint.io/getting-started), to enable ESLint to work on TypeScript code
2. Install [the `node-gyp` prerequisites for your operating system](https://github.com/nodejs/node-gyp#on-unix)

```bash
````

File: docs/guide/getting-started.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ layout: doc
...
+  
+  1. Python should be installed
+  2. Visual C++ build tools workload for Visual Studio 2022
+
+  You can use **Chocolatey** Package Manager ([See Installation / Setup](https://docs.chocolatey.org/en-us/choco/setup/))
+  ```bash
+  choco install python visualstudio2022-workload-vctools -y
+  ```
+</details>
+
```
Comment (karlhorky) [inline docs/guide/getting-started.md:22]:
```suggestion
1. Set up ESLint following [the `typescript-eslint` Getting Started docs](https://typescript-eslint.io/getting-started), to enable ESLint to work on TypeScript code
2. Install [the `node-gyp` prerequisites for your operating system](https://github.com/nodejs/node-gyp#on-unix)

```

**Review by karlhorky — CHANGES_REQUESTED** [review]
Made some comments above - I think that we can probably just link to the `node-gyp` docs, because they contain the cross-platform setup instructions which are needed here

@Newbie012 thoughts?

**Review by karlhorky — APPROVED** [review]
LGTM, thanks!

@Newbie012 over to you for final approval + merge

**Comment by Newbie012** [comment]
When I think about it, maybe it would be wise to mention node-gyp only for Windows, since you don't really need to do anything when using Linux/MacOS

**Comment by karlhorky** [comment]
> you don't really need to do anything when using Linux/MacOS

This is not true - at least not for macOS:

1. [Newer macOS doesn't have Python preinstalled](https://www.freecodecamp.org/news/how-to-fix-python-installation-errors-on-mac/#:~:text=Macs%20no%20longer%20come%20with%20Python%20pre%2Dinstalled.)
2. Xcode Command Line Tools also not pre-installed on macOS

The [`devel` docs section of packages for Ubuntu Noble](https://packages.ubuntu.com/noble/devel/) seem to indicate that GCC and make are preinstalled on Ubuntu.

But I guess macOS and Windows are going to make up the majority of users?

**Pull request merged**